src/components/DropZone.tsx

```diff
@@ -97,11 +100,13 @@ export default function DropZone({
         multiple
       />
       <button
-        className="block cursor-pointer relative py-3 px-6 text-base font-bold text-stone-700 dark:text-stone-200 bg-white dark:bg-stone-800 border-2 border-stone-700 dark:border-stone-700 rounded-lg transition-all duration-300 ease-in-out outline-none hover:shadow-md active:shadow-inner focus:shadow-outline"
```

Comment:
I appreciate the sorting, but this does make the diff a bit messy. would like to enforce this in CI rather than doing it piecemeal like this